.github/CONTRIBUTING.md

```diff
@@ -1,3 +1,18 @@
+# QCElemental
+
+QCElemental is a project collecting fundamental tools for computational 
+molecular sciences (CMS) into a lightweight Python
+interface. It is maintained by the Molecular Sciences Software
+Institute (MolSSI) and contributed to by a number of community CMS code 
+developers and users.
+
+The project welcomes new contributions and comments!
+Pull requests are tested and linted to ensure code integrity and quality.
```

Comment:
Consider "automatically tested and linted"

Reply:
The automatically is true but not sufficient. Since this is the contributing docs, may want to emphasize that you the contributor are going to have to write more tests and prove this is good new code that will benefit downstream, not just pass the existing green checks.